openapi/products/listingBuilder.yaml

```diff
@@ -0,0 +1,214 @@
+                        type: string
+                        format: uri
+                      first:
+                        type: string
+                        description: Provides a link back to the first page of results
+                        format: uri
+                      next:
+                        type: string
+                        description: The URI at which the next batch of listings can be gotten from
+                        format: uri
+    options:
+      operationId: options-businessEntityListings
+      summary: List valid HTTP verbs for /businessEntityListings
+      description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
+      responses:
+        '204':
+          description: No Content
+        '403':
+          description: User Cannot Access Entity
+        '404':
+          description: Business Not Found
+        '406':
+          description: No Listing Product Active
+      tags:
+        - Options
+        - Business Entity Listings
+    parameters: []
+components:
+  schemas:
+    businessLocationReviews:
+      title: Business Entity Listings
+      type: object
+      x-lifecycle:
+        status: trustedTester
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+        A review of an item - for example, of a restaurant, movie, or store.
+      x-tags:
+        - Business Entity Listings
+      properties:
+        name:
+          description: Public facing name of the source
+          type: string
+          example: HotFrog
+        icon:
+          description: Link to icon provided by Vendasta’s media service
+          type: string
+          format: uri
+          example: //www.cdnstyles.com/static/images/icon32/sourceId-10090.png
+        popularity:
+          description: Popularity of the source relative to other sources
+          type: number
+        sourceTypeCode:
+          description: Type of source, business directory, app (Snapchat), GPS app, etc.
+          type: string
+          example: BUSINESS_DIRECTORY
+        syncStatusCode:
+          description: The publishing status provided by the sources. Some sources provide confirmation that your data has been published (synced) other sources only provide confirmation that your data has been received (submitted)
+          type: string
+          example: SYNCED
+        listingScore:
+          description: A score based on the status of your online source
+          type: number
+        maximumlistingScore:
+          description: The maximum score this source can attain
+          type: number
+        url:
+          description: The online listing
+          format: uri
+          type: string
+        syncingError:
```

Comment:
Could do, the errors and warning each have a separate list of enums tho. Does it make sense?

Reply:
Since this is a readonly field it probably does not add to the confusion. If clients could update the status themselves that would be a good reason to keep them separate.

Reply:
I feel like it adds to my confusion if I dont have a separate set of values that are errors and a set that are warnings tho

Reply:
What does the user who is calling the API want?

Reply:
To display one type of status if there are errors, a different status if there are warnings, and the same status as only errors when there is both.